### DISCUSSION TITLE
Language Selection not applied everywhere

### DISCUSSION THREAD

**Discussion by Skgland** [community/community#77600] [Mobile] [answered]
I am in Germany with my Phone default Language configured as German, but GitHub Mobile set to English.
![Screenshot_20231202-001647.jpg](https://github.com/community/community/assets/3877590/c106970a-8afd-40f0-b390-1e0342f99d11)

But e.g. the label for merged PRs is still in German "zusammengeführt" instead of English "merged"
![Screenshot_20231202-001632.jpg](https://github.com/community/community/assets/3877590/6995fc7d-e47d-40a8-a5dc-fec0b797202f)

**Comment by iskae** [comment]
Hi @Skgland 👋 

Thanks so much for reaching out ❤️ That is unfortunate to hear, we should have loaded the correct resources respecting the in-app settings 😅 

Can you please also check the system level app language settings and tell us if `English` is selected in there too? You can check via `Settings` -> `Apps` -> `GitHub` -> `Language` 🙇 

In any case I will be creating a bug issue for us and investigate this further 🙇 

Cheers!

**Reply by Skgland** [reply]
I am on Android Version 11, so there is no per App Language Setting.

![Screenshot_20231205-151121.jpg](https://github.com/community/community/assets/3877590/05a20800-3ea0-4e81-b4de-af4fcc4bae12)

**Comment by karfau** [comment]
Here are some more examples for this, really annoying...

![Screenshot_20231213-164812_GitHub.png](https://github.com/community/community/assets/135657/467b2a11-51b3-4062-a47d-9ec7c0508389)

![Screenshot_20231213-164705_GitHub.png](https://github.com/community/community/assets/135657/45d9878f-be1a-452a-8e6c-7c63e4a8ae8b)

![Screenshot_20231213-164640_GitHub.png](https://github.com/community/community/assets/135657/78bdc8a8-9e10-45a8-a4b1-095c2f80741a)

**Comment by Koksu87PL** [comment]
Polski

**Comment by iskae** [comment, marked as answer]
Hi @Skgland 👋 

We have finally some progress on this 🎉 The issue and pull request screens should show correct translations in the upcoming releases 🎉 We will continue to work on files changed screens 🙇 

Thank you so much once again for reaching out to us and reporting this ❤️ Please stay tuned for further fixes 🙇 

Cheers!
